examples/hackernews/back-end/reactive_service/src/hackernews.service.ts

```diff
@@ -71,11 +72,16 @@ class PostsMapper {
   mapElement(
     key: number,
     it: NonEmptyIterator<Post>,
-  ): Iterable<[[number, number], Post & { upvotes: number; author: User }]> {
+  ): Iterable<[[number, number], Upvoted]> {
     const post = it.first();
     const upvotes = this.upvotes.getArray(key).length;
-    const author = this.users.getOne(post.author_id);
-    return [[[-upvotes, key], { ...post, upvotes, author }]];
+    const author = this.users.maybeGetOne(post.author_id);
+    return [
+      [[-upvotes, key], { ...post, upvotes, author }] as [
```

Comment:
The `as` shouldn't be needed here either.

Reply:
Yes